test/e2e/scalability/load.go

```diff
@@ -483,10 +486,16 @@ func GenerateConfigsForGroup(
 }
 
 func generateServicesForConfigs(configs []testutils.RunObjectConfig) []*v1.Service {
-	services := make([]*v1.Service, 0, len(configs))
+	services := make([]*v1.Service, 0)
+	currentSvcLabel := ""
 	for _, config := range configs {
+		svcLabel, found := config.GetLabelValue(svcLabelKey)
+		if !found || svcLabel == currentSvcLabel {
+			continue
+		}
+		currentSvcLabel = svcLabel
 		serviceName := config.GetName() + "-svc"
-		labels := map[string]string{"name": config.GetName()}
```

Comment:
I'd rather keep 'name' label as well. I don't remember if we read it in the load test, but I think we use it when waiting for RC to start (or sth).